PasswordRulesSharp.Cli/Commands/GenerateCommand.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.ComponentModel;
+
+using Spectre.Console;
+using Spectre.Console.Cli;
+
+using Rule = PasswordRulesSharp.Rules.Rule;
+
+namespace PasswordRulesSharp.Cli.Commands;
+
+public class GenerateCommand : Command<GenerateCommand.Settings>
+{
+    public class Settings : CommandSettings
+    {
+        [CommandOption("--rule")]
+        public string Rule { get; set; } = "";
+
+        [CommandOption("--count")]
+        [DefaultValue(5)]
+        public int Count { get; set; }
+    }
+
+    public override int Execute(CommandContext context, Settings settings)
+    {
+        var rule = Rule.FromString(settings.Rule);
+        var generator = new Generator.Generator(rule);
+
+        for (int i = 0; i < settings.Count; i++)
+            Console.WriteLine(generator.GeneratePassword());

```

Comment:
Use `AnsiConsole.MarkupLine(...)` instead since we are using Spectre.Console.

Reply:
I recently discovered this package which enables us to use `ILogger` for Spectre Console: https://github.com/lunet-io/Lunet.Extensions.Logging.SpectreConsole

Reply:
I'm unsure how to escape that. Like, what if the password contains a `[`?

Reply:
>I'm unsure how to escape that.

…by calling `.EscapeMarkup()`, duh.

Reply:
>I recently discovered this package which enables us to use ILogger for Spectre Console:

This may prove useful, but for _this_ tool, I kind of want to keep it as self-contained as possible.